.golangci.yml

```diff
@@ -0,0 +1,93 @@
+# This file was inspired by the golangci-lint one:
```

Comment:
just wondering if lint is in Makefile and get the output in a file.
currently lint action is doing nothing

Reply:
This is the golangci lint file , that is the configuration file for the linter. This is being taken into consideration when the golangcilint command is run